designsafe/static/scripts/data-depot/components/published/published-view.component.js

```diff
@@ -40,6 +40,12 @@ class PublishedViewCtrl {
             });
         };
 
+        if (this.$stateParams.filePath.endsWith(projId + "/")) {
```

Comment:
Would the case ever arise where there'd be a filePath that ends with the projId, but no "/"?

Reply:
Ah I see, what about something like `this.$stateParams.filePath.replace('/', '').endsWith(projId)` or is that kinda hacky?